godot-core/src/builtin/plane.rs

```diff
@@ -217,7 +217,14 @@ impl Plane {
     /// Returns a normalized copy of the plane.
     #[inline]
     pub fn normalized(self) -> Self {
-        Plane::new(self.normal.normalized(), self.d)
+        let length: real = self.normal.length();
+        if length == 0.0 {
+            return Plane {
+                normal: Vector3::ZERO,
+                d: 0.0,
+            };
+        }
```

Comment:
Is this behavior actually useful? The resulting plane is geometrically degenerate, and C++/GDScript only does that because they don't have a proper error handling mechanism (panic).

On the other hand, `Vector2::normalize()` also returns the zero vector in such a case, so this would be consistent. So maybe we can keep it (but document what happens in this case).